packages/react-devtools-shared/src/inspectedElementMutableSource.js

```diff
@@ -62,8 +62,15 @@ export function inspectElement({
   rendererID: number,
 |}): Promise<InspectElementReturnType> {
   const {id} = element;
+
+  // This could indicate that the DevTools UI has been closed and reopened.
+  // The in-memory cache will be clear but the backend still thinks we have cached data.
+  // In this case, we need to tell it to resend the full data.
+  const forceFullData = !inspectedElementCache.has(id);
```

Comment:
nit: i wonder if a name like `resetCachedElement` or similar would convey more precisely the intent of this argument? i feel like `forceFullData` seems a little vague: which data? what does it mean that it's full?

Reply:
just kind of thinking out loud: is there any instance where we *wouldn't* want the backend to send back the full data even when the element isn't cached in the frontend?

Reply:
> is there any instance where we wouldn't want the backend to send back the full data even when the element isn't cached in the frontend?

No, I don't think so.

The term "full data" comes from the types of response the backend sends:
* hydrated-path
* *no-change
* *not-found
* full-data